tibiawikisql/models/item.py

```diff
@@ -282,15 +282,18 @@ def from_article(cls, article):
             if attribute in item._raw_attributes and item._raw_attributes[attribute]:
                 item.attributes.append(ItemAttribute(item_id=item.article_id, name=name,
                                                      value=item._raw_attributes[attribute]))
-        item._parse_attributes()
-        item._parse_resists()
-        vocations = item._raw_attributes.get('vocrequired')
-        if vocations and "none" not in vocations.lower():
-            vocation = vocations.replace('and', '+').replace(',', '+').replace(' ', '')
-            item.attributes.append(ItemAttribute(item_id=item.article_id, name="vocation", value=vocation))
-        item._parse_sounds()
-        item._parse_store_value()
-        return item
+        try:
+            item._parse_attributes()
+            item._parse_resists()
+            vocations = item._raw_attributes.get('vocrequired')
+            if vocations and "none" not in vocations.lower():
+                vocation = vocations.replace('and', '+').replace(',', '+').replace(' ', '')
+                item.attributes.append(ItemAttribute(item_id=item.article_id, name="vocation", value=vocation))
+            item._parse_sounds()
+            item._parse_store_value()
+            return item
+        except Exception:
```

Comment:
FYI parsing was failing for [Knightly Bed](https://tibia.fandom.com/wiki/Knightly_Bed) and [Knightly Bench](https://tibia.fandom.com/wiki/Knightly_Bench) due to missing Tibia Coins amount.

I've edited the articles in TibiaWiki and now it should be working. Nonetheless, I think it is good to still catch the exception here 👍